### PR TITLE
[OCPBUGS-34875]: Adding note about hosted clusters to API server certificate docs

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OCPBUGS-34875

Link to docs preview: https://80148--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/certificates/api-server.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

File: security/certificates/api-server.adoc [hunk 1]
```diff
@@ -11,4 +11,9 @@ cluster CA. Clients outside of the cluster will not be able to verify the
 API server's certificate by default. This certificate can be replaced
 by one that is issued by a CA that clients trust.
 
+[NOTE]
+====
+In hosted control plane clusters, self-signed certificates from the API cannot be replaced.
```
Comment (bergerhoffer) [inline security/certificates/api-server.adoc:16]: Maybe consider more active?

```suggestion
In hosted control plane clusters, you cannot replace self-signed certificates from the API.
```

**Review by bergerhoffer — COMMENTED** [review]
Just a quick thought.

Also just making sure you know that you're allowed to merge your own PRs (as long as you get peer review first)

File: security/certificates/api-server.adoc (hunk 1 above)
Comment (lahinson) [reply security/certificates/api-server.adoc:16]: Good suggestion! And thanks for the reminder that I merge my own work after it's been peer reviewed 😄

**Pull request merged**